arithmetization/src/main/java/net/consensys/linea/zktracer/module/bin/Bin.java

```diff
@@ -77,7 +66,7 @@ public List<ColumnHeader> columnsHeaders() {
   }
 
   @Override
-  public int lineCount() {
-    return this.chunks.lineCount();
+  public StackedSet<BinOperation> operations() {
```

Comment:
This is a getter, could be better to make it like this : getOperations(). The comment works for all other modules.

Reply:
done